### PR TITLE
feat(settings): Add back up seed phrase banner in profile settings

### PR DESCRIPTION
**Pull request by borismelnik**
Closes: #5743

### What does the PR do

Add back up seed phrase banner in profile settings

### Affected areas

Settings

### Screenshot of functionality

Снимок экрана 2022-06-03 в 13 50 10

### CODE REVIEW

**Review by PascalPrecht — APPROVED** [review]
Change-wise, this looks good to me. Just one question:

I think there was discussion around this banner to show up in the entire app, not just the settings section, am I correct, or am I confusing that with another banner?

**Comment by saledjenic** [comment]
@borismelnik yes, as @PascalPrecht said, that banner line should be on the app global level (it will be use for many different notifications, not only for not backed up seed phrase.

File: src/app/global/local_account_sensitive_settings.nim [hunk 1]
```diff
@@ -96,6 +96,8 @@ const LSS_KEY_IS_DDMMYY_DATE_FORMAT* = "is_DDMMYY_date_format"
 const DEFAULT_IS_DDMMYY_DATE_FORMAT = false
 const LSS_KEY_IS_24H_TIME_FORMAT* = "is_24h_time_format"
 const DEFAULT_IS_24H_TIME_FORMAT = false
+const LSS_KEY_USER_DECLINED_BACKUP_BUNNER* = "userDeclinedBackupBanner"
```
Comment (noeliaSD) [inline src/app/global/local_account_sensitive_settings.nim:99]: Typo B**A**NNER?? In both constants!

File: ui/app/AppLayouts/Profile/stores/ProfileStore.qml [hunk 2]
```diff
@@ -13,6 +13,13 @@ QtObject {
     property string ensName: userProfile.preferredName || userProfile.firstEnsName || userProfile.ensName
     property string profileLargeImage: userProfile.largeImage
     property string icon: userProfile.icon
+    property bool userDeclinedBackupBanner: localAccountSensitiveSettings.userDeclinedBackupBanner
+    onUserDeclinedBackupBannerChanged: {
+        print(userDeclinedBackupBanner !== localAccountSensitiveSettings.userDeclinedBackupBanner, userDeclinedBackupBanner, localAccountSensitiveSettings.userDeclinedBackupBanner)
```
Comment (noeliaSD) [inline ui/app/AppLayouts/Profile/stores/ProfileStore.qml:18]: Maybe leftover?? :)

**Review by noeliaSD — APPROVED** [review]
LGTM! Just minor comments!

File: src/app/global/local_account_sensitive_settings.nim (hunk 1 above)
Comment (PascalPrecht) [reply src/app/global/local_account_sensitive_settings.nim:99]: Could make it a Rabbits cohort easter egg and call it `BUNNY` instead

**Comment by borismelnik** [comment]
> Change-wise, this looks good to me. Just one question:
> 
> I think there was discussion around this banner to show up in the entire app, not just the settings section, am I correct, or am I confusing that with another banner?

@PascalPrecht I read it `creating an account and going to the profile section should show the banner` like `going to the profile section should show the banner`. @saledjenic @PascalPrecht guys are you sure about? if yes I will fix it.

**Comment by saledjenic** [comment]
>  @saledjenic @PascalPrecht guys are you sure about? if yes I will fix it.

@borismelnik yes, I am pretty sure, as that banner line should be used for other app notifications, like... "new update available", "you're in the wallet test mode", "backup your seed phrase", but at the end we can sort that out when we're dealing with activity center bell icon, as it should be alway visible (regardless to the displayed app's section).

**Comment by borismelnik** [comment]
> > @saledjenic @PascalPrecht guys are you sure about? if yes I will fix it.
> 
> @borismelnik yes, I am pretty sure, as that banner line should be used for other app notifications, like... "new update available", "you're in the wallet test mode", "backup your seed phrase", but at the end we can sort that out when we're dealing with activity center bell icon, as it should be alway visible (regardless to the displayed app's section).

I think its should be done in another issue. Fixing every each banner in app is out of issue scope. But we can make an issue and assign it to me and I will add it :)

**Pull request merged**